### PR TITLE
improve grabbing cursor perf (fixes #3790)

### LINKED ISSUE #3790: Don't set grabbing class on the body. Causing 2ms style recalculations.
**Description:**

For look-controls, setting class on the body on each click, causing 2ms style recalculations. We should just set the style directly.

### PR DESCRIPTION
**Pull request by krzysztof-grzybek**
Fixes #3790

Hi @ngokevin , I added changes, but I didn't spot any performance improvement (I checked it in Chrome performance devtools). Maybe I didn't understand something?

### CODE REVIEW

File: src/components/look-controls.js [hunk 1]
```diff
@@ -259,12 +258,34 @@ module.exports.Component = registerComponent('look-controls', {
     }
   },
 
+  /**
+   * Shows grabbing cursor on scene
+   */
+
```
Comment (ngokevin) [inline src/components/look-controls.js]:
```suggestion

```

File: src/components/look-controls.js [hunk 2]
```diff
@@ -259,12 +258,34 @@ module.exports.Component = registerComponent('look-controls', {
...
+      'cursor: -moz-grabbing;' +
+      'cursor: -webkit-grabbing;';
+    var newStyle = currentStyle.replace(/;$/, '') + ';' + cursorStyle;
+    this.el.sceneEl.canvas.setAttribute('style', newStyle);
+  },
+
+  /**
+   * Hides grabbing cursor on scene
+   */
+
```
Comment (ngokevin) [inline src/components/look-controls.js]:
```suggestion

```

File: src/components/look-controls.js [hunk 3]
```diff
@@ -259,12 +258,34 @@ module.exports.Component = registerComponent('look-controls', {
     }
   },
 
+  /**
+   * Shows grabbing cursor on scene
+   */
+
+  showGrabbingCursor: function () {
+    var currentStyle = this.el.sceneEl.canvas.getAttribute('style') || '';
```
Comment (ngokevin) [inline src/components/look-controls.js]: Just this:

```js
this.el.sceneEl.canvas.style.cursor = 'grabbing';
this.el.sceneEl.canvas.style.mozCursor = 'grabbing';
this.el.sceneEl.canvas.style.webkitCursor = 'grabbing';
```

**Comment by ngokevin** [comment]
I saw in devtools taking time for "Apply Style Changes".

File: src/components/look-controls.js (hunk 3 above)
Comment (krzysztof-grzybek) [reply src/components/look-controls.js]: I'm afraid it won't work.. Take a look, what has to be prefixed is css property value, not property name. On the other hand, are You sure that those prefixes are needed? I tested it on latest chrome and firefox and it works without prefix, also take a look [here](https://caniuse.com/#search=grabbing).

Comment (ngokevin) [reply src/components/look-controls.js]: Yeah can just get rid of prefixes. It's not that big a feature anyways.

**Pull request merged**